frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
```

Comment:
move line 3 and 4 down after the newline

Reply:
Resolved in commit 6c4688c5a5c1853aa8ff09a0d49fdb749f87bdbb